examples/toolbar/toolbar.html

```diff
@@ -44,32 +51,123 @@ <h1>Toolbar Example</h1>
     <h2 id="ex_label">Example</h2>
     <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
     <div id="ex1">
-        <div class="toolbar" role="toolbar">
-            <div class="toolbar-item selected" role="button" tabindex="0">
-                Tool 1
+      <div class="format" role="toolbar" aria-label="Text Formatting" aria-controls="textarea1">
+          <div class="group characteristics">
+            <div role="button"
+                class="item bold"
+                aria-pressed="false"
+                aria-label="bold"
+                value="bold"
```

Comment:
This is not an html button. From an HTML perspective, this is a div. therefore yu can't use the value attribute. You could instead use an HTML button. That would be better anyway because we should use as much native HTML as possible. Only use the ARIA necessary to demonstrate ARIA toolbars and the ariety of elements  they can contain.